.eslintrc.json

```diff
@@ -29,7 +29,7 @@
     "new-cap": ["error", {
       "capIsNewExceptions": ["DynamicTable"]
     }],
-    "no-console": ["error", {
+    "no-console": ["warn", {
```

Comment:
What does this change do?

Reply:
https://eslint.org/docs/rules/no-console
The change makes "npm run compile" not output an error but just a warning when a developer is using console.log somewhere in the frontend. I think it's probably best but thoughts?

Reply:
I think it should stay an error, warnings get ignored. We should in general try be trying to make static checks stricter, not weaker over time. A leftover `console.log` is likely a debugging statement that was left behind that we shouldn't be depending on review to catch if it can be caught by Travis proactively.

Fix this and I'll merge it (assuming it's still passing Travis), since no one's going to realistically be able to test/review all of LORIS for this PR and it's better that it goes in so we catch any bugs before the last minute..